test/polytopes.jl

```diff
@@ -6,6 +6,9 @@
     s = Segment(Point(1.0,), Point(2.0,))
```

Comment:
```suggestion
    s = Segment(P2(1.0,), P2(2.0,))
```

Reply:
@juliohm This is not a `P2`. This is a 1D segment.